### PR TITLE
adds Journeys, OfficeMax #1609 #1640

### PR DESCRIPTION
**Pull request by jonahadkins**

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -28715,13 +28717,22 @@
...
-    "match": ["shop/stationery|OfficeMax"],
     "tags": {
-      "brand": "Office Max",
       "name": "Office Max",
       "shop": "stationery"
     }
   },
+  "shop/stationery|OfficeMax": {
+    "count": 115,
+    "match": ["shop/stationery|OfficeMax"],
```
Comment (bhousel) [inline config/canonical.json]: I think what you're trying to do here is match "Office Max" with a space..
(the `match` property contains the alternative names that we don't prefer)

```suggestion
    "match": ["shop/stationery|Office Max"],
```

File: config/canonical.json [hunk 2]
```diff
@@ -28715,13 +28717,22 @@
   },
   "shop/stationery|Office Max": {
```
Comment (bhousel) [inline config/canonical.json]: and this "Office Max" gets removed since it is matched on another entry

**Comment by matkoniecz** [comment]
Something went wrong, Github diff shows that it reverts many recent changes. I suspect that it may be a Github problem.

**Comment by bhousel** [comment]
yeah the merge in 5aa4055 removed some things it wasn't supposed to..

I ended up manually making the change in bef2100

**Pull request closed without merging**